src/App.js

```diff
@@ -16,6 +16,7 @@ import SignIn from "./pages/signin.js";
 import Dashboard from "./pages/shelterDashboard.js";
 import Pet from "./pages/pet.js";
 import FullScreenLoader from './components/FullScreenLoader';
+import { Header } from "react-bootstrap/lib/Modal.js";
```

Comment:
Looks like you are just importing the wrong `Header` component here.

Reply:
That is not my code

I am using import Header from './components/Header/Header.js'